config/sanctum.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Stateful Domains
+    |--------------------------------------------------------------------------
+    |
+    | Requests from the following domains / hosts will receive stateful API
+    | authentication cookies. Typically, these should include your local
+    | and production domains which access your API via a frontend SPA.
+    |
+    */
+
+    'stateful' => explode(',', env('SANCTUM_STATEFUL_DOMAINS', 'localhost,127.0.0.1,127.0.0.1:8000,::1')),
```

Comment:
This is fine, but remember to add your local webapp host to the .env file under SANCTUM_STATEFUL_DOMAINS. Example: `SANCTUM_STATEFUL_DOMAINS="myfoobartest.local,localhost,127.0.0.1"`

Reply:
I left the default config as-is.

Reply:
Yup, might be worth a note though, because requests from domains that are not in the list won't receive any cookies (even if they resolve to 127.0.0.1).